src/refinement-list/__tests__/refinement-list.spec.ts

```diff
@@ -36,6 +37,34 @@ describe('RefinementList', () => {
     expect(fixture).toMatchSnapshot();
   });
 
+  it('should pass attribute, limit, showMoreLimit, operator sortBy to connector', () => {
+    const createWidget = jest.spyOn(
+      NgAisRefinementList.prototype,
+      'createWidget'
+    );
+    const fixture = createRenderer({
+      defaultState,
+      template: `<ais-refinement-list 
+          attribute="brands"
+          [limit]="20"
+          [showMoreLimit]="30"
+          operator="and"
+          sortBy="name"
+          ></ais-refinement-list>`,
+      TestedWidget: NgAisRefinementList,
+      additionalDeclarations: [NgAisHighlight, NgAisFacetsSearch],
+    })({});
+    expect(createWidget).toHaveBeenCalledWith(connectRefinementList, {
+      attributeName: 'brands',
+      limit: 20,
+      showMoreLimit: 30,
+      operator: 'and',
+      sortBy: 'name',
+      escapeFacetValues: true,
+    });
+    createWidget.mockRestore();
+  });
+
```

Comment:
I could help split this into 6 small tests but I didn't think it was worth it
let me know what you think.

Reply:
IMO the more the test is focused the better it is.

Reply:
ok, so you would suggest writing a separate `it` for each attribute?